GeoEntity.md

```diff
@@ -94,8 +94,7 @@ All responses with an HTTP status of 200 will consist of a JSON object with a to
           "lat": 64.2008413,
           "lng": -149.4936733
         },
-        "location_type": "APPROXIMATE",
-        "viewport": {
+	"viewport": {
```

Comment:
looks like your indentation got messed up slightly on this line. (possibly on other sample jsons too?)